app_dart/test/request_handlers/override_github_build_status_test.dart

```diff
@@ -0,0 +1,240 @@
+// Copyright 2019 The Flutter Authors. All rights reserved.
```

Comment:
Any reasons to use 2019 instead of 2020 in all the new files?

Reply:
Just that I copied and pasted. I can update the dates, but in other repos we just try to keep the dates consistent from the earliest date in the repo.